data/content.go

```diff
@@ -697,6 +697,12 @@ var ContentHeuristics = map[string]*Heuristics{
 			regexp.MustCompile(`(?m)^(\s*namespace\s*[\w\.]+\s*{|\s*\/\/)`),
 		),
 	},
+	".csc": &Heuristics{
+		rule.Or(
+			rule.MatchingLanguages("GSC"),
+			regexp.MustCompile(`(?m)^\s*#\s*(?:using|insert|include|define|namespace)[ \t]+\w|^\s*(?>(?:autoexec|private)\s+){0,2}function\s+(?>(?:autoexec|private)\s+){0,2}\w+\s*\(|\b(?:level|self)[ \t]+thread[ \t]+(?:\[\[[ \t]*(?>\w+\.)*\w+[ \t]*\]\]|\w+)[ \t]*\([^\r\n\)]*\)[ \t]*;|^[ \t]*#[ \t]*(?:precache|using_animtree)[ \t]*\(`),
```

Comment:
@lafriks .. I got failure for this one in particular.
Check this section of `Readme`. 
https://github.com/go-enry/go-enry#divergences-from-linguist
<img width="1267" alt="Screenshot 2022-03-04 at 2 33 15 PM" src="https://user-images.githubusercontent.com/29868154/156732965-5425f6b7-25a3-4e08-90b3-16d1ec891a79.png">